templates/docs/patterns/contextual-menu/accessibility.md

```diff
@@ -0,0 +1,35 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Contextual menu | Components
+---
+
+## How it works
+
+The contextual menu is a secondary menu which can be applied to any button. The button contains a few aria-attributes:
+
+- An `aria-controls` attribute matching the `id` of the span containing the menu.
+- An `aria-expanded` attribute, the value always being the opposite of the `aria-hidden` value on the span containing the menu.
+- An `aria-haspopup` with the value of true.
+- Keyboard interaction:
+  - The Enter and space keys opens the menu
+  - The Escape closes the menu
+  - When the menu is open, the Tab key will move through the menu items and once it leaves the final item, the menu closes. AT will then announce the pop up has collapsed.
+- Touch interaction:
+  - Double tap opens and closes menu
```

Comment:
Is it double tap or just tap? I don't think we implement anything specific for mobile, so simple click/tap opens and closes the menu, right?

Reply:
Thanks - I missed this! Either I wrote this months ago or maybe Marina added it as a suggestion